sites/docs/src/content/docs/guidelines/pipelines/recommendations/publication_credit.md

```diff
@@ -4,16 +4,30 @@ subtitle: Recommended recognition for nf-core pipeline publications
 weight: 123
 ---
 
-Pipeline publications should acknowledge the nf-core community (e.g. in an academic journal).
+## Acknowledging nf-core in Academic Publications
 
-**At a minimum**, the nf-core community should be thanked in the acknowledgment section. A good example is from the [nf-core/clipseq](https://doi.org/10.12688/wellcomeopenres.19453.1)) preprint:
+If your work has utilized nf-core pipelines or elements of the framework, please ensure that you cite the following publication, which describes the community and framework in detail:
```

Comment:
I think this changes the scope of this page significantly.

The previous text read _Pipeline publications_ - specifically publications about nf-core pipelines.

Acknowledgements, co-authorship and the rest is excessive if someone is just running an nf-core pipeline for their analysis.

Reply:
hmmm not sure i understand. co-authorship and others were also mentioned previously...I just made them into sub-sections. are you saying to just to make them less obvious or remove them all together ?

Reply:
I think what Phil means is this page is aimed at papers publishing an nf- core pipeline. So announcing to the world a particular pipeline exists. 

By rephrasing to 'utilising' a pipeline, it now is saying if you use any nf-core pipeline to analyse data, you also have to cite the main nf-core paper and acknowledge the community, which is asking a bit much .

At least this is how I'm reading it and I think I'm getting the same impression as Phil

Reply:
ah i c u now

Reply:
Yup, that ☝🏻

Reply:
ya reverted back most of that language to keep it minmal